packages/frontend/src/config/envParsers.js

```diff
@@ -0,0 +1,19 @@
+export const envValIsSet = (envVal) => typeof envVal === "string";
+
+export const parseBooleanFromShell = (envVal) =>
+    envValIsSet(envVal) ? envVal === "yes" || envVal === "true" : undefined;
+
+export const parseObjectFromShell = (envVal) => {
+    if (envValIsSet(envVal)) {
+        try {
+            let parsed = JSON.parse(envVal);
+            return parsed;
```

Comment:
No need to assign this to a value, you can return the expression directly in the `try`.